include/crow/http_response.h

```diff
@@ -78,6 +78,13 @@ namespace crow
         {
             *this = std::move(r);
         }
+        
+        response(std::string contentType, std::string body)
+        {
+            set_header("Content-Type",mime_types[contentType]);
+            this->body = body;
```

Comment:
might be best to use `response(std::string contentType, std::string body) : body(std::move(body))` instead.

Also please add a ctor that includes a response code as well

Reply:
Thanks for the review, made the requested changes.